site/src/components/css-display/AccordionView.tsx

```diff
@@ -0,0 +1,193 @@
+  "indicator"
+);
+
+const fontSize: string[] = new Array("fontSize", "minHeight");
+const fontWeight: string[] = new Array("fontWeight");
+const cursor: string[] = new Array("cursor");
+const letters: string[] = new Array("letterSpacing");
+const border: string[] = new Array(
+  "borderStyle",
+  "borderWidth",
+  "outlineWidth",
+  "outlineStyle"
+);
+const text: string[] = new Array("fontStyle");
+const outline: string[] = new Array("outline");
+const shadow: string[] = new Array("shadow");
+const align: string[] = new Array("textAlign");
+const textDecoration: string[] = new Array("textDecoration");
+const transform: string[] = new Array("textTransform");
+const fontFamily: string[] = new Array("fontFamily");
+const lineHeight: string[] = new Array("lineHeight");
+
+const BlockView: React.FC<{ name: string }> = ({ name }) => {
+  switch (true) {
+    case color.some((c) => name.includes(c)):
+      console.log(name);
+      return <ColorBlock hideToken colorVar={name} />;
+    case fontSize.some((fs) => name.includes(fs)):
+      return <FontSizeBlock hideToken fontSize={name} />;
+    case fontWeight.some((fw) => name.includes(fw)):
+      return <FontWeightBlock hideToken fontWeight={name} />;
+    case cursor.some((fw) => name.includes(fw)):
+      return <CursorBlock hideToken cursor={name} />;
+    case letters.some((fw) => name.includes(fw)):
+      return <LetterSpacingBlock hideToken letterSpacing={name} />;
+    case border.some((fw) => name.includes(fw)):
+      return <LineBlock hideToken token={name} lineStyle={name} />;
+    case text.some((fw) => name.includes(fw)):
+      return <TextBlock fontSize={name} />;
+    case outline.some((fw) => name.includes(fw)):
+      return <OutlineBlock outline={name} />;
+    case shadow.some((fw) => name.includes(fw)):
+      return <ShadowBlockCell shadowVar={name} />;
+    case align.some((fw) => name.includes(fw)):
+      return <TextBlock textAlign={name} />;
+    case textDecoration.some((fw) => name.includes(fw)):
+      return <TextBlock textDecoration={name} />;
+    case transform.some((fw) => name.includes(fw)):
+      return <TextBlock textTransform={name} />;
+    case fontFamily.some((fw) => name.includes(fw)):
+      return <TextBlock fontFamily={name} />;
+    case lineHeight.some((fw) => name.includes(fw)):
+      return <TextBlock lineHeight={name} />;
+    default:
+      return <p>{name}</p>;
+  }
+};
+
+export const CssAccordionView: React.FC<{ value: string }> = ({ value }) => {
+  const [cssVariablesData, setCssVariablesData] =
+    useState<CssVariableData | null>(null);
+
+  useEffect(() => {
+    const fetchJsonData = async () => {
+      try {
+        const data = require("./cssVariables.json");
+        setCssVariablesData(data);
+      } catch (err) {
+        console.error("Error fetching JSON data:", err);
+      }
+    };
+
+    fetchJsonData();
+  }, []);
+
+  if (!cssVariablesData) {
+    return <Banner>Failed to load characteristic data</Banner>;
```

Comment:
It's good that we're displayed an error message when there's no data but I can also see this message for a couple seconds when the page is loading so maybe we also need a loading state?